src/tests/src/io/yml-importers/test_full.cpp

```diff
@@ -117,16 +117,16 @@ BOOST_AUTO_TEST_CASE(test_full)
         - port: injection_port
           field: flow
           definition: generation
+      binding-constraints:
+        - id: balance
+          expression: injection_port.flow = 0
```

Comment:
This expression is probably invalid, try
```cpp
sum_connections(injection_port.flow) = 0
```

Reply:
sum_connection isn't handled by the convertorVisitor